projects/js-packages/publicize-components/src/components/social-previews/modal.js

```diff
@@ -72,7 +72,7 @@ export default withSelect( select => {
 	// Use the featured image by default, if it's available.
 	let image = featuredImageId ? getMediaSourceUrl( getMedia( featuredImageId ) ) : '';
 
-	const sigSettings = getImageGeneratorPostSettings();
+	const sigSettings = getImageGeneratorSettings();
```

Comment:
I think the use of Post in the name was to distinguish it from the blog/plugin settings. As long as we don't think it could be confusing then I think it's fine to remove.

Reply:
Reverted to `getImageGeneratorPostSettings`